--- install_flatc.sh	2024-03-28 14:24:36.381826031 +0100
+++ install_flatc_mod.sh	2024-03-28 14:24:10.938980158 +0100
@@ -27,7 +27,7 @@
 # Prints the flatbuffers version of the git submodule.
 print_flatbuffers_version(){
     pushd "${FLATBUFFERS_PATH}" > /dev/null
-    git describe --tags "$(git rev-list --tags --max-count=1)" | sed 's/^v//'
+    git describe --tags | sed 's/^v//'
     popd > /dev/null
 }
 
