--- original.py	2024-10-02 16:17:30.285667189 +0200
+++ pte_to_header.py	2024-10-02 16:17:49.121595621 +0200
@@ -59,7 +59,7 @@
 if __name__ == "__main__":
     args = parser.parse_args()
     outfile = os.path.join(args.outdir, args.outfile)
-    attr = f'__attribute__((section("{args.section}"), aligned(16))) char '
+    attr = f'__attribute__((aligned(16))) char '
 
     with open(args.pte, "rb") as fr, open(outfile, "w") as fw:
         data = fr.read()
@@ -75,4 +75,4 @@
         fw.write(hexstring)
         print(
             f"Input: {args.pte} with {len(data)} bytes. Output: {outfile} with {len(hexstring)} bytes. Section: {args.section}."
-        )
\ No newline at end of file
+        )
