--- pte_to_header.py	2024-04-02 09:47:49.410452634 +0200
+++ executorch/examples/arm/executor_runner/pte_to_header.py	2024-04-02 09:50:05.573382064 +0200
@@ -59,7 +59,7 @@
 if __name__ == "__main__":
     args = parser.parse_args()
     outfile = os.path.join(args.outdir, args.outfile)
-    attr = f'__attribute__((section("{args.section}"), aligned(16))) char '
+    attr = f'__attribute__((aligned(16))) char '
 
     with open(args.pte, "rb") as fr, open(outfile, "w") as fw:
         data = fr.read()
