--- targets.bzl	2024-03-11 09:48:36.671482385 +0100
+++ targets-modified.bzl	2024-03-11 09:45:39.017529274 +0100
@@ -103,6 +103,15 @@
     )
 
     executorch_generated_lib(
+        name = "generated_lib_all_ops",
+        deps = [
+            ":executorch_all_ops",
+            "//executorch/kernels/portable:operators",
+        ],
+        **generated_lib_common_args
+    )
+
+    executorch_generated_lib(
         name = "generated_lib_aten",
         deps = [
             ":executorch_aten_ops",
@@ -120,3 +129,4 @@
         ],
         define_static_targets = True,
     )
+
